tests/sentry/api/path_params/resources/test_organization_slug.py

```diff
@@ -0,0 +1,378 @@
+from typing import Any
+from unittest.mock import MagicMock, patch
+
+from pytest import fixture
+from rest_framework.request import Request
+
+from sentry.testutils.cases import TestCase
+from sentry.testutils.helpers.features import with_feature
+
+from .test_id_or_slug_path_params_mixin import APIIdOrSlugTestMixin
+
+
+class OrganizationSlugTests(TestCase, APIIdOrSlugTestMixin):
+    @fixture(autouse=True)
+    def _mock_subdomain_is_region(self):
+        with patch("sentry.api.bases.organization.subdomain_is_region"):
+            yield
+
+    @patch("sentry.types.region.subdomain_is_region")
+    def organization_test(self, endpoint_class, slug_params, *args):
+        slug_kwargs = {param: self.slug_mappings[param].slug for param in slug_params}
+        id_kwargs = {param: self.slug_mappings[param].id for param in slug_params}
+
+        request = Request(request=self.make_request())
```

Comment:
nit: put in a `setUp`

Reply:
I didn't put the request in a setUp because some of the endpoints,  I decide to mock the request.